assets/src/application/ui/layout/descriptionList/styles.scss

```diff
@@ -0,0 +1,15 @@
+.ee {
+	&-description-list-grid {
```

Comment:
It's a bit tricky to make this grid dynamic.
What I would like to implement in a separate PR is to define `grid-template-rows` based on some passed prop or to compute somehow it's value.

Reply:
try this for the parent container (dl?)

```css
		display: grid;
		gap: var(--ee-margin-big);
		grid-template-columns: repeat(auto-fit, minmax(SET_PX_WIDTH_YOU_WANT, 1fr));
		grid-auto-rows: SET_PX_HEIGHT_YOU_WANT;
```

gotta say:

```csss
.ee {
	&-
```

this is very clever namespacing 👍

Reply:
updated:

![image](https://user-images.githubusercontent.com/1477453/77069475-29f24700-69f1-11ea-961e-282cadc69c1b.png)

***
will have to tweak more when registry will be coupled